wrap/FakeItEasy/project.json

```diff
@@ -0,0 +1,16 @@
+{
+  "version": "99.99.99-wrapped",
+  "dependencies": {
+        "Castle.Core": "4.0.0-alpha001"
+  },
+  "frameworks": {
+    "netstandard1.6": {
+      "wrappedProject": "../../src/FakeItEasy.dotnet/FakeItEasy.dotnet.csproj",
+      "bin": {
+        "assembly": "../../src/FakeItEasy.dotnet/bin/{configuration}/FakeItEasy.dll",
+        "pdb": "../../src/FakeItEasy.dotnet/{configuration}/FakeItEasy.pdb"
+      },
+      "imports": [ "dotnet5.4" ]
+    }
+  }
+}

```

Comment:
Currently the PCL output assembly cannot be directly consumed by the project.json/xproj based projects. They can only reference project.json based projects or nuget packages.  This wraps the output so it can be used by the .NET Core unit test project.